webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerToggle.js

```diff
@@ -21,7 +21,7 @@ class NotificationDrawerToggle extends Component {
     switch (actionType) {
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({
-          count: NotificationsStore.getNotifications().length,
+          unreadNotificationsCount: NotificationsStore.hasUnreadNotifications(),
```

Comment:
I'm a bit puzzled how the tests results did not reflect the missing argument on this function?